static/locales/en.json

```
@@ -20,9 +22,52 @@
+    "live2dPreview": "Live2D Preview",
+    "cardInfoPreview": "Card Info",
+    "tagsPreview": "Tags",
+    "notesPreview": "Notes",
+    "selectModelToPreview": "Please select a model to preview",
+    "characterCardsList": "Character Cards List",
+    "selectModel": "Select Model",
+    "selectVoice": "Select Voice",
+    "scanModels": "Scan Models",
+    "scanningModels": "Scanning models...",
+    "scanningVoices": "Scanning voices...",
+    "scanError": "Scan failed",
```

Biome (2.1.2)

[error] The key scanError was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)